test/demo/compute_profile.yml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: localhost
+  tasks:
+  - name: "Create a compute profile"
+    foreman_compute_profile:
+      username: "admin"
+      password: "changeme"
+      server_url: "https://foreman.example.com"
+      verify_ssl: false
+      name: "Enterprise"
+      state: "present"
+    foreman_compute_profile:
```

Comment:
I forgot a line here:

`- name: "Remove the compute profile"`

Reply:
Line has been added. ;)